pkg/security/podsecuritypolicy/util/util.go

```diff
@@ -170,3 +171,52 @@ func UserFallsInRange(id int64, rng extensions.UserIDRange) bool {
 func GroupFallsInRange(id int64, rng extensions.GroupIDRange) bool {
 	return id >= rng.Min && id <= rng.Max
 }
+
+// PSPAllowsHostVolumePath is a utility for checking if a PSP allows the host volume path.
+// This only checks the path. You should still check to make sure the host volume fs type is allowed.
+func PSPAllowsHostVolumePath(psp *extensions.PodSecurityPolicy, hostPath string) bool {
```

Comment:
Is it mandatory to use PSP in the function name? I don't see a reason for that. `psputil.AllowsHostVolumePath()` looks good to me either.